static/app/cloudfoundry.js

```diff
@@ -76,6 +76,23 @@
                 });
         };
 
+        // Get quota usage data
+        this.getQuotaUsage = function(org) {
+            $http.get('/v2/organizations/' + org.guid + '/memory_usage')
```

Comment:
I think the deep nesting can be solved with using promises a little better:

```
var toReturn = org,
      scope = {};
$http.get('/v2/organizations/' + org.guid + '/memory_usage')
  .then(function(response) {
    toReturn.used_memory = response.data.memory_usage_in_mb;
    return $http.get('/v2/organizations/' + org.guid);
  }).then(function(response) {
    return $http.get(response.data.entity.quota_definition_url);
  }).then(function(response) {
    toReturn.quota = response.data;
    toReturn.quota.entity.used_memory = toReturn.used_memory;
  }).catch(function(error) {
    // Do nothing
  });
```

I didn't get to test that this actually worked, but I got some of the ideas from here: http://solutionoptimist.com/2013/12/27/javascript-promise-chains-2/. I'm also not a promise expert by any means, so somebody else might have even better ideas.